cicecore/cicedynB/general/ice_init.F90

```diff
@@ -2868,39 +2884,55 @@ subroutine set_state_var (nx_block, ny_block, &
          ! ice concentration/thickness
          !---------------------------------------------------------
 
-         if (trim(ice_data_type) == 'box2001'    .or. &
-             trim(ice_data_type) == 'smallblock' .or. &
-             trim(ice_data_type) == 'channel'    .or. &
-             trim(ice_data_type) == 'bigblock'   .or. &
-             trim(ice_data_type) == 'blockep5'   .or. &
-             trim(ice_data_type) == 'uniformp5'  .or. &
-             trim(ice_data_type) == 'gauss') then
+         if (trim(ice_data_conc) == 'p5') then
```

Comment:
All of these many options are under the ice_ic='default' case, which is a little confusing.  I'd like to look at the logic of how all these options interact - maybe ice_ic options just need to be renamed to be a little clearer what 'default' means.

Reply:
I added ice_ic='internal' as an option and that is equivalent to default now.